Makefile

```diff
@@ -87,59 +85,64 @@ test_unit:
 test_race:
 	@VERSION=$(VERSION) go test -mod=readonly -race $(PACKAGES_NOSIMULATION)
 
-test_sim_app_nondeterminism:
+test_sim_nondeterminism:
 	@echo "Running nondeterminism test..."
 	@go test -mod=readonly $(SIMAPP) -run TestAppStateDeterminism -Enabled=true -v -timeout 10m
 
-test_sim_app_custom_genesis_fast:
+test_sim_custom_genesis_fast:
 	@echo "Running custom genesis simulation..."
 	@echo "By default, ${HOME}/.gaiad/config/genesis.json will be used."
 	@go test -mod=readonly $(SIMAPP) -run TestFullAppSimulation -Genesis=${HOME}/.gaiad/config/genesis.json \
 		-Enabled=true -NumBlocks=100 -BlockSize=200 -Commit=true -Seed=99 -Period=5 -v -timeout 24h
 
-test_sim_app_fast:
```

Comment:
@mircea-c why was this deleted?

Reply:
either this was accidental or it wasn't removed from the `PHONY`

Reply:
Just missed removing it from the phony. `test_sim_multi_seed_short` accomplishes the same thing.